src/common/task.c

```diff
@@ -182,6 +182,9 @@ task_spec *alloc_nil_task_spec(task_id task_id) {
 }
 
 int64_t task_spec_size(task_spec *spec) {
+  if (spec == NULL) {
```

Comment:
Oops, thanks for catching this! I was originally allocating an empty task using this, but turned out to not need it.